### PR TITLE
Bratseth/rank type information 2 take 2

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Comment by bratseth** [comment]
@lesters please review

**Comment by lesters** [comment]
Question: These models are copied into config-model/src/test/cfg/application/ml_models. The same models are in ../ml_serving. These directories are modified as far as I can see during the actual unit tests, which is why this is done, but as we do have a few copies lying around in different parts of the source tree could these tests be merged in some way?

File: config-model/src/test/java/com/yahoo/vespa/model/ml/MlModelsTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                     4, model.rankProfileList().getRankProfiles().size());
+
+        RankProfilesConfig.Builder builder = new RankProfilesConfig.Builder();
+        model.getSearchClusters().get(0).getConfig(builder);
+        RankProfilesConfig config = new RankProfilesConfig(builder);
+        assertEquals(3, config.rankprofile().size());
+        assertEquals("test", config.rankprofile(2).name());
+        RankProfilesConfig.Rankprofile.Fef test = config.rankprofile(2).fef();
+
+        // Compare string content in a denser for that config:
```
Comment (lesters) [inline config-model/src/test/java/com/yahoo/vespa/model/ml/MlModelsTest.java]: denser?

**Review by lesters — APPROVED** [review]
LGTM!

File: config-model/src/test/java/com/yahoo/vespa/model/ml/MlModelsTest.java (hunk 1 above)
Comment (bratseth) [reply config-model/src/test/java/com/yahoo/vespa/model/ml/MlModelsTest.java]: "format"

**Comment by bratseth** [comment]
Yes, we could uniquefy them, but why?

**Pull request merged**